packages/relay-compiler/codegen/RelayCodegenRunner.js

```diff
@@ -247,14 +247,19 @@ class RelayCodegenRunner {
         this.parserWriters[parserName].forEach(writer =>
           dependentWriters.push(writer),
         );
-        if (!this.parsers[parserName]) {
-          // have to load the parser and make sure all of its dependents are set
-          await this.parseEverything(parserName);
-        } else {
-          this.parseFileChanges(parserName, files);
+        try{
```

Comment:
nit: space between `try` and `{`. Also, I don't know if GitHub is misrendering this but the indenting looks a bit off.